bindings/python/crocoddyl/fwd.hpp

```diff
@@ -10,9 +10,6 @@
 #define BINDINGS_PYTHON_CROCODDYL_CORE_FWD_HPP_
 
 #include <eigenpy/eigenpy.hpp>
-#include <boost/python.hpp>
-#include <boost/python/enum.hpp>
-#include <boost/python/scope.hpp>
```

Comment:
Is there a particular reason why we do this? It seems this might trigger compilation issues.

Reply:
From what I can tell, `boost/python.hpp` has included both `enum.hpp` and `scope.hpp` [as far back as 1.60](https://www.boost.org/doc/libs/1_60_0/boost/python.hpp), so there's not need to include them.

Meanwhile, eigenpy's `fwd.hpp` includes `boost/python.hpp` itself [as far back as tag 2.7.10](https://github.com/stack-of-tasks/eigenpy/blob/v2.7.10/include/eigenpy/fwd.hpp) which is the version required in the CMakeLists file, so there's no need to include it.
 
I think in the absolute eigenpy should handle including these headers itself.

Reply:
Ok, thanks for the explanation.